resolwe_bio/tools/genehcluster.py

```diff
@@ -115,7 +115,7 @@
         'gene_symbols': calculated_genes_dict,
         'order': dend['leaves'],
         'zero_gene_symbols': list_zero_genes,
-        'zero_sample_ids': [args.sampleids[index] for index in sample_zero]
+        'zero_sample_ids': [int(args.sampleids[index]) for index in sample_zero]
```

Comment:
I'm not sure, but I think these are already integers. It's the [test](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tests/files/sample_cluster_data.json.gz) [files](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tests/files/gene_cluster_data.json.gz) that have strings.